### DISCUSSION TITLE
🌐 Quaver Localization Project (QLP)

### DISCUSSION THREAD

**Discussion by GreenScreen410** [Quaver/Quaver#4766] [Feature Request]
### Is your feature request related to a problem? Please describe.

Add official translation.

### Describe the solution you'd like

Players will have a great local play experience.

### Describe alternatives you've considered

Currently, Quaver has players of a wide variety of nationalities. But some of them can speak English, and others don't. As there are players of various nationalities, I hope there will be a localization project for a great play experience in the country. There are already many services specialized in localization projects such as Crowdin. With these services, we can set up a translation team. I'll collect all the strings in the game in a few months. Can this project go ahead?

**Comment by Reihynn** [comment]
I would like to help with localization 👍👍

**Comment by ItzKoshepen** [comment]
I would also be up for localization for traditional Chinese

**Comment by WilliamQiufeng** [comment]
I could do simplified chinese :D

**Comment by HanHan3z** [comment]
I could do Simplified Chinese.

**Comment by GreenScreen410** [comment]
Crowdin page: https://crowdin.com/project/quaver

**Comment by GreenScreen410** [comment]
Discord: https://discord.gg/MYva4XUVvQ

**Comment by EpiCreeper** [comment]
id like to help translate to hebrew, but quaver doesnt support the hebrew alphabet

**Comment by GreenScreen410** [comment]
I want to make 2 roles; Translator and Proofreader.

### Translator
- Everyone can join localization project.
- Use crowdin, vote good texts and wait for proofreader's check.
- If there's some active and good translator, We can promote proofreader.

### Proofreader (QLP Team members, I want to include those who have expressed their intention to participate so far.)
- Only trusted person
- Have profile badge (I'm not sure this is okay)
- Check translator's text.

**Comment by ry0shi** [comment]
I could certainly contribute to a Russian localisation 🙃

**Comment by GreenScreen410** [comment]
I think this project will be very long term project; Stay tuned!

**Comment by AiAe** [comment]
We've uploaded most of what we want the game to have translated.

Everyone can request to join, I will approve once I see them.
https://crowdin.com/project/quaver

If language is missing that someone wants to translate to let me know, but for now those that I have added should be enough.
